<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeUI</class>
 <widget class="QMainWindow" name="RangeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabControls">
       <attribute name="title">
        <string>Controls</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Comms</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblEndPoint">
             <property name="text">
              <string>System Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leEndPoint"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnConnect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpSetup">
          <property name="title">
           <string>Setup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpControl">
          <property name="title">
           <string>Control</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btnStart">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStatus">
             <property name="text">
              <string>Get Status</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnResults">
             <property name="text">
              <string>Get Results</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTerminal">
       <attribute name="title">
        <string>Raw Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblTerminal">
          <property name="text">
           <string>Raw Input:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="txtTerminal"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
